server/lib/sightingsController.js

```diff
@@ -11,7 +11,16 @@ const logger = require('./logger.js');
 // Get all sightings in the database
 exports.getSightings = (searchOpts = {}) => {
 	logger.info('Getting Sightings...');
-	logger.info(`Using some variables so lint doesn't scream: ${searchOpts}`);
+	const query = {};
+
+	logger.info(Object.keys(searchOpts));
+
+	if (searchOpts.id) { // This can be reused for various searchOpts - currently id only
```

Comment:
This isn't going to scale. You're setting it up to only search by one thing. What if you want to find only cats on desks that you've spotted?

I'd pull out the query builder into a separate function that generates the whole query json (and can be tested ;) )

Reply:
It'd end up being an if statement for each option. We can pull it into a separate function to enable testing.